README.md

```diff
@@ -59,6 +59,8 @@ You can start contributing by adding the following:
 | [Voyage AI](https://www.voyageai.com/) | [Link](https://docs.voyageai.com/reference/embeddings-api) | Y | Voyage AI provides API endpoints for embedding and reranking models. |
 | [AI/ML API](https://aimlapi.com/) | [Link](https://docs.aimlapi.com/) | Y |  An API aggregator that provides access to 100+ AI models via a single API. |
 | [Wit.ai](https://wit.ai/) | [Link](https://wit.ai/docs/http/20240304/) | Y | Wit.ai provides APIs to build natural language experiences. |
+| [Amazon Web Services (AWS) AI](https://aws.amazon.com/machine-learning/ai-services/) | [Link](https://docs.aws.amazon.com/machine-learning/index.html) | Y | AWS AI offers a wide range of AI services, including NLP, image and video analysis, and machine learning. |
```

Comment:
Add API docs link .. not the aws docs

Reply:
is this correct https://aws.amazon.com/solutions/ai/intelligent-document-processing/